doc/conf.py

```diff
@@ -95,6 +95,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx.ext.linkcode',
+    'sphinx.ext.inheritance_diagram',
```

Comment:
An attempt to fix 358 errors like `/home/jovyan/repos/private/panel/doc/api/panel.__main__.rst:4: ERROR: Unknown directive type "inheritance-diagram".`

Reply:
Hmm, that should have been disabled, I actually don't like the inheritance diagrams, but I'll re-enable for now.